pyogrio/geopandas.py

```diff
@@ -37,33 +39,56 @@ def _stringify_path(path):
     return path
 
 
-def _try_parse_datetime(ser):
+def _try_parse_datetime(ser, datetimes="UTC"):
     import pandas as pd  # only called when pandas is known to be installed
+    from pandas.api.types import is_string_dtype
+
+    datetimes = datetimes.upper()
+    datetime_kwargs = {}
+    if datetimes == "STRING":
+        if not is_string_dtype(ser.dtype):
+            res = ser.astype("string").str.replace(" ", "T")
```

Comment:
When is this case used? For when we use a GDAL version that does not support returning as string (for the arrow code path?)

Reply:
Yes indeed, GDAL < 3.11 didn't support returning strings for arrow...